doc/index.md

```diff
@@ -0,0 +1,25 @@
+---
+# https://vitepress.dev/reference/default-theme-home-page
+layout: home
+
+hero:
+  name: "wire"
+  text: "a tool to deploy nixos systems"
+  # tagline: My great project tagline
+  actions:
+    - theme: brand
+      text: Read Guide
+      link: /guide/wire
+    - theme: alt
+      text: Reference
+      link: /reference/cli
+
+features:
+  - title: Parallelism
+    details: Build and deploy many nodes at once
+  - title: Secrets & Keys
+    details: Fast keys that never enter your store
```

Comment:
don't forget about my nitpicks above, also you may want to change `details` of the 2nd element to simply just `Manage your secrets your way` or something, communicate to the reader that wire is unopinionated in secret management, and while not decrypted and in common usecases, secrets *can* be added to the store if the user uses flakes and keeps their secrets along with their configuration, as the entire flake is copied to the store.